history/taskana-simplehistory-provider/src/test/java/configuration/TaskanaEngineConfigurationTest.java

```diff
@@ -17,7 +18,8 @@ class TaskanaEngineConfigurationTest extends AbstractAccTest {
   void testCreateTaskanaEngine() throws Exception {
     DataSource ds = DataSourceGenerator.getDataSource();
     TaskanaEngineConfiguration taskEngineConfiguration =
-        new TaskanaEngineConfiguration(ds, false, false, DataSourceGenerator.getSchemaName());
+        new TaskanaEngineConfigurationBuilder(ds, false, false, DataSourceGenerator.getSchemaName())
+            .build();
```

Comment:
Just name that class `Builder`? 
Without the static import we would have `new TaskanaEngineConfiguration.Builder()` which I prefer.